services/admin/src/server.ts

```diff
@@ -782,7 +826,14 @@ export async function start({
   logger = new Logger(LogSource.VxAdminService),
   port = PORT,
   workspace,
-}: Partial<StartOptions>): Promise<void> {
+}: Partial<StartOptions>): Promise<Server> {
+  const auth = new DippedSmartCardAuthWithMemoryCard({
+    card: new WebServiceCard({ baseUrl: 'http://localhost:3001' }),
```

Comment:
Good q! We're in a way already hardcoding if you look at the proxies for frontend communication with the smartcards service, e.g. https://github.com/votingworks/vxsuite/blob/5cf6e3799241734a67f63b87663cd081b416ad73/frontends/election-manager/prodserver/setupProxy.js#L18

So that makes me feel confident that doing so on the backend is safe as well. From a code cleanliness perspective, a config var probably still makes sense, but this code path will be shortlived, so doesn't feel worth the trouble